src/InputWithLabel.jsx

```diff
@@ -0,0 +1,27 @@
+import { useRef, useEffect } from 'react';
+
+
+function InputWithLabel({ children, value, onChange}) {
+    const inputRef = useRef(null);
+    useEffect(() => {
+        inputRef.current.focus();
+    }, []);
```

Comment:
If you delete the empty dependency array, the autofocus will work not only when you add todo but when you delete it as well. Besides for now it works only when you add todo using key enter, not the add button.

Reply:
fixed